addons/project/static/src/js/project_list.js

```diff
@@ -39,14 +39,44 @@ const ProjectListController = ListController.extend({
         return this._super(...arguments);
     },
 
-    _stopRecurrence(recurringResIds, resIds, mode) {
+    _countRecords(recurrenceIds, resIds) {
+        return this._rpc({
+            model: 'project.task',
+            method: 'read_group',
+            args: [
+                [['recurrence_id', 'in', recurrenceIds], ['id', 'not in', resIds]],
+                ['ids:count(id)'],
+                ['recurrence_id']
+            ],
+        });
+    },
+
+    async _stopRecurrence(recurringResIds, resIds, mode) {
+        const recurrenceIds = [];
+        for (const record of this.getSelectedRecords()) {
+            const recurrenceId = record.data.recurrence_id;
+            if (recurrenceId && !recurrenceIds.includes(recurrenceId)) {
+                recurrenceIds.push(recurrenceId);
+            }
+        }
+        // list recurrences that have tasks left after deleting/archiving
+        let countsLeft = await this._countRecords(recurrenceIds, recurringResIds);
+        countsLeft = countsLeft.map(rec => rec.recurrence_id[0]);
+        // so we check that no recurrence is absent, as it would mean no task is left
+        const allowContinue = recurrenceIds.every(rec => countsLeft.includes(rec));
+
         let warning;
         if (resIds.length > 1) {
-            warning = _t('It seems that some tasks are part of a recurrence.');
+            warning = allowContinue
+            ? _t('It seems that some tasks are part of a recurrence.')
+            : _t('It seems that some tasks are part of a recurrence. At least one of them must be kept as a model to create the next occurences.');
```

Comment:
You can add a tabulation here. 🙂 
```suggestion
                ? _t('It seems that some tasks are part of a recurrence.')
                : _t('It seems that some tasks are part of a recurrence. At least one of them must be kept as a model to create the next occurences.');
```